### ISSUE TITLE
Segmentation of celebA datasets

### ISSUE THREAD

**Issue by Destinyrhx** [tsingqguo/misf#24]
Regarding the division of the celebA dataset, in your experience, how many images are suitable for the test set and the validation set?

**Comment by Li-Xiaoguang** [comment]
For the CelebA dataset, we use the standard train, validation, and test set. You can get the corresponding image lists from the official website.
